src/update/UpdatePopupDialog.cpp

```diff
@@ -64,6 +65,10 @@ UpdatePopupDialog::UpdatePopupDialog (wxWindow* parent, const VersionPatch& vers
     Layout();
     Fit();
     Center();
+
+    wxSize sz = GetSize();
+    SetSizeHints(sz.x, std::min(sz.y, 600),
+        wxDefaultCoord, wxDisplay().GetGeometry().GetHeight() - 100);
```

Comment:
Maybe instead
```
const auto maxHeight = wxDisplay().GetGeometry().GetHeight() - 100;
const auto minHeight = std::min({ sz.y, 600, maxHeight });
SetSizeHints(sz.x, minHeight, wxDefaultCoord, maxHeight);
```

just to be sure your minimum height is not more than the maximum